### DISCUSSION TITLE
How to set compatibility with SAP field format of type date

### DISCUSSION THREAD

**Discussion by garuda64** [dbosoft/YaNco#275] [Q&A]
Hello again, how can I establish the sending format of a date field so that it is compatible with the definition of the field in SAP (dd.MM.yyyy).

![imagen](https://github.com/dbosoft/YaNco/assets/17913367/37177aac-052c-405a-9530-c9c5d66ad91c)

**Comment by garuda64** [comment]
I greatly appreciate your support :)

**Comment by fw2568** [comment]
YaNco contains a (configureable) FieldMapper, that maps from DateTime to SAP date, time or timestamp automatically. 

So you just have to use `SetField("DATE", <your .NET Date Time Type>)`
